examples/viewer.py

```diff
@@ -908,6 +1207,333 @@ def next_frame() -> None:
+    self.curr_object = rigid_object_manager.add_object_by_template_handle(
+        object_template_handle
+    )
+
+    # Agent local coordinate system is Y up and -Z forward.
+    # Move object above table surface, then turn on Kinematic mode
+    set_object_state(self, self.curr_object, position, rotation_quaternion)
+    self.simulating = False
+    self.curr_angle_rotated_degrees = 0.0
+
+    # reset axis that the new object rotates around when being displayed
+    self.object_rotation_axis = ObjectRotationAxis.Y
+
+    # for some reason they all end in "_:0000" so remove that substring before print
+    obj_name = self.curr_object.handle.replace("_:0000", "")
+
+    # print out object name and its index into the list of the objects
+    # in dataset.
+    print("")
+    logger.info(
+        f'Command: placing object "{obj_name}" from template handle index: {index}\n'
+    )
+
+
+def set_object_state(
+    self,
+    obj,
+    position=HabitatSimInteractiveViewer.DEFAULT_OBJ_POSITION,
+    rotation=HabitatSimInteractiveViewer.DEFAULT_OBJ_ROTATION,
+) -> None:
+    # Set ManagedBulletRigidObject transform in world space
+    obj.translation = position
+    obj.rotation = rotation
+
+
+def rotate_displayed_object(
+    self, obj, degrees_per_sec=HabitatSimInteractiveViewer.ROTATION_DEGREES_PER_SEC
+) -> None:
+
+    # How much to rotate current ManagedBulletRigidObject this frame
+    delta_rotation_degrees = degrees_per_sec * Timer.prev_frame_duration
+
+    # check for a full 360 degree revolution
+    # and clamp to 360 degrees if we have passed it
+    reset_curr_rotation_angle = False
+    if self.curr_angle_rotated_degrees + delta_rotation_degrees >= 360.0:
+        reset_curr_rotation_angle = True
+        delta_rotation_degrees = 360.0 - self.curr_angle_rotated_degrees
+
+    if self.object_rotation_axis == ObjectRotationAxis.Y:
+        # rotate about object's local y axis (up vector)
+        y_rotation_in_degrees = mn.Deg(delta_rotation_degrees)
+        y_rotation_in_radians = mn.Rad(y_rotation_in_degrees)
+        obj.rotate_y_local(y_rotation_in_radians)
+    else:
+        # rotate about object's local x axis (horizontal vector)
+        x_rotation_in_degrees = mn.Deg(delta_rotation_degrees)
+        x_rotation_in_radians = mn.Rad(x_rotation_in_degrees)
+        obj.rotate_x_local(x_rotation_in_radians)
+
+    # update current rotation angle
+    self.curr_angle_rotated_degrees += delta_rotation_degrees
+
+    # reset current rotation angle if it passed 360 degrees
+    if reset_curr_rotation_angle:
+        self.curr_angle_rotated_degrees = 0.0
+
+        # change ManagedBulletRigidObject's axis of rotation
+        if self.object_rotation_axis == ObjectRotationAxis.Y:
+            self.object_rotation_axis = ObjectRotationAxis.X
+        else:
+            self.object_rotation_axis = ObjectRotationAxis.Y
+
+
+def write_video_file(self) -> None:
+    # write each PIL Image in self.video_frames to file one at a time
```

Comment:
Can you make the docstrings in all the functions consistent

Reply:
Sorry, what do you mean by this?

Reply:
Yes